docs/server/source/server-reference/configuration.md

```diff
@@ -102,6 +110,18 @@ The settings with names of the form `database.*` are for the database backend
 * `database.replicaset` is only relevant if using MongoDB; it's the name of the MongoDB replica set, e.g. `bigchain-rs`.
 * `database.connection_timeout` is the maximum number of milliseconds that BigchainDB will wait before giving up on one attempt to connect to the database backend.
 * `database.max_tries` is the maximum number of times that BigchainDB will try to establish a connection with the database backend. If 0, then it will try forever.
+* `database.ssl` is a flag that determines if BigchainDB connects to the
+  backend database over SSL or not. This can be set to any value (the presence of a value
+  qualifies as set or true in this case) to enable SSL connections. Do not
+  specify this parameter to connect over plaintext connection.
```

Comment:
I noticed that the default value of `ssl` is `false`. According to the above, "the presence of a value qualifies as set or true in this case" which just leaves me very confused; it implies that setting it to anything means TRUE (including setting it to `true`). I think what is needed here is some concrete examples of possible real-world values for this setting.